### PR TITLE
Added extra documentation for Windows Audit Policies in Syscheck whodata

### PR DESCRIPTION
**Pull request by jotacarma90**
## Description
Hey team, this PR is to go deeper into the Windows whodata section of syscheck.
Detailing more the configurations that are necessary for the correct functioning of whodata.
It also improved the description of how this module works, explaining how Wazuh can detect these changes and act accordingly, reconfiguring directories in real-time mode.
Improved section of Manual configuring Audit Policies.

This closes issue https://github.com/wazuh/wazuh-documentation/issues/5668.

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst [hunk 1]
```diff
@@ -11,23 +11,22 @@ Auditing who-data in Windows
...
-These changes produce audit events that are processed by *syscheck* and reported to the manager. This feature is only compatible with systems greater than Windows Vista.
-
-Configuration
-^^^^^^^^^^^^^
+The who-data monitoring functionality uses the Microsoft Windows auditing subsystem to get the related information about who made modifications in a monitored directory. These changes produce audit events that are processed by *syscheck* and reported to the manager. This feature is only compatible with systems greater than Windows Vista.
 
-To enable the Whodata feature, the SACL of the directory to be monitored must be properly configured. Wazuh automatically performs this task when 
-the tag ``whodata="yes"`` is declared within the ``directories`` statement in the ``ossec.conf`` file:
+For the correct operation of this auditing subsystem, it is necessary to have configured some Local Audit Policies and the SACLs of each configured directory from which to receive these events. Wazuh automatically performs this task when the tag ``whodata="yes"`` is declared within the ``directories`` statement in the ``ossec.conf`` file:
```
Comment (chemamartinez) [inline source/user-manual/capabilities/auditing-whodata/who-windows.rst:16]:
```suggestion
For the correct operation of the who-data feature, it is necessary to have configured some Local Audit Policies as well as the SACLs of each monitored directory. Wazuh automatically performs this task when the tag ``whodata="yes"`` is declared within the ``directories`` statement in the ``ossec.conf`` file:
```

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst [hunk 2]
```diff
@@ -11,23 +11,22 @@ Auditing who-data in Windows
...
 
 .. code-block:: xml
 
     <syscheck>
       <directories check_all="yes" whodata="yes">C:\Windows\System32\drivers\etc</directories>
     </syscheck>
 
-System audit policies also need to be properly configured. This part is also automatically done for most supported Windows systems. If your system is
-superior to Windows Vista but the audit policies cannot be self-configured, see :ref:`the guide to configure Local Audit Policies<who-windows-policies>`.
+However, other processes could modify some of these configurations, preventing syscheck from receiving the necessary information. In this case the directories configured in whodata mode will be reconfigured to realtime mode, so that syscheck will continue to generate alerts. Wazuh detects these changes in two different ways:
```
Comment (chemamartinez) [inline source/user-manual/capabilities/auditing-whodata/who-windows.rst:24]:
```suggestion
Even though FIM configures the needed Local Audit Policies and SASLs when launched. Other services may change this configuration preventing who-data from receiving the monitored events. In this case, FIM detects this configuration change and switches all the directories configured with who-data to real-time mode. The two available mechanisms to detect these configuration changes are:
```

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst [hunk 3]
```diff
@@ -11,23 +11,22 @@ Auditing who-data in Windows
...
+
+#. Windows generates specific events (ID 4719) when one of the Audit Policies is modified (Success removed).
+#. Periodically, Wazuh checks that the Audit Policies and the SACLs are configured as expected. It is possible to modify this interval, see :ref:`the configuration section of windows_audit_interval<reference_ossec_syscheck_windows_audit_interval>`.
```
Comment (chemamartinez) [inline source/user-manual/capabilities/auditing-whodata/who-windows.rst:27]:
```suggestion
#. Periodically, Wazuh checks that the Audit Policies and the SACLs are configured as expected. It is possible to modify the frequency of this verification with :ref:`windows_audit_interval<reference_ossec_syscheck_windows_audit_interval>`.
```

**Review by chemamartinez — CHANGES_REQUESTED** [review]

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst (hunk 1 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows.rst:16]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst (hunk 2 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows.rst:24]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst (hunk 3 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows.rst:27]: Done

**Review by chemamartinez — APPROVED** [review]

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 4]
```diff
@@ -15,31 +18,45 @@ to activate the capture of successful events. You can do it from the Local Group
...
 .. thumbnail:: ../../../images/whodata/audit-policies-old-method.png
     :title: Audit policies configuration (old method)
     :align: center
     :width: 60%
+
+SACLs in Windows
+^^^^^^^^^^^^^^^^
+
+A system access control list (SACL) enables administrators to log attempts to access a secured object.
+SACLs of each directory can be checked and modified through its Properties, Security tab, and clicking on Advanced:
```
Comment (javimed) [inline source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:43]:
```suggestion
You can check and modify SACLs of each directory through **Properties**, selecting the **Security** tab, and clicking on **Advanced**:
```

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 5]
```diff
@@ -15,31 +18,45 @@ to activate the capture of successful events. You can do it from the Local Group
...
+
+.. thumbnail:: ../../../images/whodata/windows-security-setting.png
+    :title: Windows advanced security settings
+    :align: center
+    :width: 60%
+
+In the Auditing tab it is necessary to have a Success entry:
```
Comment (javimed) [inline source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:50]:
```suggestion
In the Auditing tab it's necessary to have a *Success* entry:
```

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 6]
```diff
@@ -15,31 +18,45 @@ to activate the capture of successful events. You can do it from the Local Group
...
+
+.. thumbnail:: ../../../images/whodata/sacl-configuration.png
+    :title: Auditing entry success
+    :align: center
+    :width: 60%
+
+It can be created by adding these advanced permissions:
```
Comment (javimed) [inline source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:57]:
```suggestion
You can create it by adding these advanced permissions:
```

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/ossec-conf/syscheck.rst [hunk 7]
```diff
@@ -971,7 +971,7 @@ For more information, please read :ref:`auditing who-data <auditing-whodata>`
 windows_audit_interval
 ----------------------
 
-Sets the frequency in seconds with which the Windows agent will check that the SACLs of the directories monitored in whodata mode are correct.
+Sets the frequency in seconds with which the Windows agent will check that the Local Audit Policies and the SACLs of the directories monitored in whodata mode are correct.
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/reference/ossec-conf/syscheck.rst:974]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst [hunk 8]
```diff
@@ -11,23 +11,22 @@ Auditing who-data in Windows
 How it works
 ^^^^^^^^^^^^
 
-The who-data monitoring functionality uses the Microsoft Windows auditing subsystem to get the related information about who made modifications in a monitored directory.
-These changes produce audit events that are processed by *syscheck* and reported to the manager. This feature is only compatible with systems greater than Windows Vista.
-
-Configuration
-^^^^^^^^^^^^^
+The who-data monitoring functionality uses the Microsoft Windows auditing subsystem to get the related information about who made modifications in a monitored directory. These changes produce audit events that are processed by *syscheck* and reported to the manager. This feature is only compatible with systems greater than Windows Vista.
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows.rst:14]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst [hunk 9]
```diff
@@ -11,23 +11,22 @@ Auditing who-data in Windows
...
 
-To enable the Whodata feature, the SACL of the directory to be monitored must be properly configured. Wazuh automatically performs this task when 
-the tag ``whodata="yes"`` is declared within the ``directories`` statement in the ``ossec.conf`` file:
+For the correct operation of the who-data feature, it is necessary to have configured some Local Audit Policies as well as the SACLs of each monitored directory. Wazuh automatically performs this task when the tag ``whodata="yes"`` is declared within the ``directories`` statement in the ``ossec.conf`` file:
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows.rst:16]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst [hunk 10]
```diff
@@ -11,23 +11,22 @@ Auditing who-data in Windows
...
 
 .. code-block:: xml
 
     <syscheck>
       <directories check_all="yes" whodata="yes">C:\Windows\System32\drivers\etc</directories>
     </syscheck>
 
-System audit policies also need to be properly configured. This part is also automatically done for most supported Windows systems. If your system is
-superior to Windows Vista but the audit policies cannot be self-configured, see :ref:`the guide to configure Local Audit Policies<who-windows-policies>`.
+Even though FIM configures the needed Local Audit Policies and SASLs when launched, other services may change this configuration preventing who-data from receiving the monitored events. In this case, FIM detects this configuration change and switches all the directories configured with who-data to real-time mode. The two available mechanisms to detect these configuration changes are:
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows.rst:24]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst [hunk 11]
```diff
@@ -11,23 +11,22 @@ Auditing who-data in Windows
...
+
+#. Windows generates specific events (ID 4719) when one of the Audit Policies is modified (Success removed).
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows.rst:26]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows.rst [hunk 12]
```diff
@@ -11,23 +11,22 @@ Auditing who-data in Windows
...
+#. Periodically, Wazuh checks that the Audit Policies and the SACLs are configured as expected. It is possible to modify the frequency of this verification with :ref:`windows_audit_interval<reference_ossec_syscheck_windows_audit_interval>`.
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows.rst:27]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 13]
```diff
@@ -1,12 +1,15 @@
...
+
 .. _who-windows-policies:
 
-Manual configuration of the Local Audit Policies in Windows
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Manual configuration of the Windows Audit Policies
+==================================================
+
+Local Audit Policies in Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:12]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 14]
```diff
@@ -15,31 +18,45 @@ to activate the capture of successful events. You can do it from the Local Group
 
     gpedit.msc
 
-- `Advanced Audit Policy Configuration section method`_
-- `Audit Policy section method`_
-
-Advanced Audit Policy Configuration section method
---------------------------------------------------
+Go to ``Security Settings -> Advanced Audit Policy Configuration`` and configure to Success the following policies:
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:21]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 15]
```diff
@@ -15,31 +18,45 @@ to activate the capture of successful events. You can do it from the Local Group
...
 
-Recommended option to configure policies. You have to activate the following options:
-
-1) Object Access -> File System
-2) Object Access -> Handle Manipulation
+1) Object Access -> Audit File System
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:23]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 16]
```diff
@@ -15,31 +18,45 @@ to activate the capture of successful events. You can do it from the Local Group
...
+2) Object Access -> Audit Handle Manipulation
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:24]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 17]
```diff
@@ -15,31 +18,45 @@ to activate the capture of successful events. You can do it from the Local Group
...
     :width: 60%
 
-Audit Policy section method
----------------------------
-
-This option is only recommended if the previous method cannot be followed because
-your host is Windows Vista or Windows Server 2008. To do this, edit the following policy:
-
-Security Settings -> Local Policies -> Audit Policy -> Audit object access
+These policies are part of the Audit Object Access category, so another possibility is to configure the entire category to Success in ``Security Settings -> Local Policies -> Audit Policy``,
```
[the remark this answers is not in the document]
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:31]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst (hunk 5 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:50]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst (hunk 6 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:57]: Done

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 18]
```diff
@@ -15,31 +18,44 @@ to activate the capture of successful events. You can do it from the Local Group
...
     :width: 60%
 
-Audit Policy section method
----------------------------
-
-This option is only recommended if the previous method cannot be followed because
-your host is Windows Vista or Windows Server 2008. To do this, edit the following policy:
-
-Security Settings -> Local Policies -> Audit Policy -> Audit object access
+If your system doesn't allow configuring subcategories through *Advanced Audit Policy Configuration*, you can configure to *Success* the entire category. These policies are part of the *Audit object access* category. Go to to **Security Settings** -> **Local Policies** -> **Audit Policy** and configure this category.
```
Comment (javimed) [inline source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:31]:
```suggestion
If your system doesn't allow configuring subcategories through *Advanced Audit Policy Configuration*, you can configure to *Success* the entire category. These policies are part of the *Audit object access* category. Go to **Security Settings** -> **Local Policies** -> **Audit Policy** and configure this category.
```

**Review by javimed — APPROVED** [review]

File: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst [hunk 19]
```diff
@@ -15,31 +18,44 @@ to activate the capture of successful events. You can do it from the Local Group
...
-Security Settings -> Local Policies -> Audit Policy -> Audit object access
+If your system doesn't allow configuring subcategories through *Advanced Audit Policy Configuration*, you can configure to *Success* the entire category. These policies are part of the *Audit object access* category. Go to **Security Settings** -> **Local Policies** -> **Audit Policy** and configure this category.
 
 .. thumbnail:: ../../../images/whodata/audit-policies-old-method.png
     :title: Audit policies configuration (old method)
     :align: center
     :width: 60%
+
+SACLs in Windows
+^^^^^^^^^^^^^^^^
```
Comment (javimed) [inline source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst:39]:
```suggestion
----------------
```

**Review by javimed — APPROVED** [review]

**Pull request merged**